File: resourcespoet/src/test/java/com/commit451/resourcespoet/DrawableTest.java
package com.commit451.resourcespoet;

import org.junit.Test;

/**
 * Tests the resources creation
 */
public class DrawableTest {

    @Test
    public void drawableTest() throws Exception {
        ResourcesPoet poet = ResourcesPoet.create()
                .addDrawable("logo", "@drawable/logo");

        TestUtil.assertEquals("drawable.xml", poet);
    }
}

File: src/test/kotlin/com/commit451/resourcespoet/DrawableTest.kt
package com.commit451.resourcespoet

import org.junit.Test

/**
 * Tests the resources creation
 */
class DrawableTest {

    @Test
    fun drawableTest() {
        val poet = ResourcesPoet.create()
            .addDrawable("logo", "@drawable/logo")

        TestUtil.assertEquals("drawable.xml", poet)
    }
}
